docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -81,7 +86,17 @@ content-type: *
 * `expo-server-defined-headers` is an [Expo SFV](expo-sfv.md) dictionary. It defines headers that a client library MUST store until overwritten by a newer dictionary, and they MUST be included in every subsequent [manifest request](#manifest-request).
 * `cache-control` - A value of `cache-control: private, max-age=0` is recommended to ensure the newest manifest is returned. Setting longer cache ages could result in stale updates.
 * `content-type` MUST be determined by _proactive negotiation_ as defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.4.1). Since the client library is [required](#manifest-request) to send an `accept` header with each manifest request, this will always be either `application/expo+json`, `application/json`; otherwise the request would return a `406` error.
-
+* `expo-signature` SHOULD contain the signature of the manifest to be used during the validation step of [code signing](#code-signing) if the request for the manifest contained the `expo-expects-signature` header.
+* `expo-asset-headers` MAY contain a JSON dictionary of header (key, value) pairs to include with asset requests. This is specified as a header so that dynamic headers needed to request assets (HMACs for example) don't alter the `expo-signature`. This is structured as follows:
```

Comment:
Might be nice to mention authorization in general:
```suggestion
* `expo-asset-headers` MAY contain a JSON dictionary of header (key, value) pairs to include with asset requests. This is specified as a header so that dynamic headers needed to request assets (HMACs or authorization tokens, for example) don't alter the `expo-signature`. This is structured as follows:
```

Reply:
Also, disallow "host"? (case-insensitive)